src/popup/locales/en.json

```diff
@@ -872,7 +870,7 @@
       "to": "To",
       "hash": "Transaction Hash",
       "messageUrlOwner": "Message the URL owner (optional)",
-      "scanAddress": "Scan {protocolName} address",
+      "scanAddress": "Scan the recipient’s account address QR code in order to send {protocolName} to them.",
```

Comment:
I feel that the `protocolName` don't fit the sentence. For the aeternity the sentence would be "...in order to send aeternity to them". We are sending AE Coin, not aeternity itself :)